catalog/USGS/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS.jsonnet

```diff
@@ -0,0 +1,327 @@
+    least squares linear regression. The slope represents the average percent cover change per 
+    year over the times-series and the p-value reflects the confidence of change in each pixel.
+    The structural change method partitions the time-series into segments of similar slope values,
+    with statistically significant break-points indicating perturbations to the prior trajectory. 
+    The break point trends analysis suite relies on structural break methods, resulting in the 
+    identification of the number and timing of breaks in the time-series, and the significance of
+    each segment. We produce the following statistics: 1) for each component, each year, the 
+    presence/absence of breaks, 2) the slope, p-value, and standard error of the segment occurring
+    in each year, 3) the overall model R2 (quality of model fit to the temporal profile), and 
+    4) an index, Total Change Intensity. This index reflects the total amount of change occurring
+    across components in that pixel. The linear and structural change methods generally agreed on 
+    patterns of change, but the latter found breaks more often, with at least one break point in 
+    most pixels. The structural change model provides more robust statistics on the significant 
+    minority of pixels with non-monotonic trends, while detrending some interannual signal 
+    potentially superfluous from a long-term perspective. Trends products can be downloaded from the
+    [Multi-Resolution Land Characteristics Consortium](https://www.mrlc.gov/data).
+
+    See also:
+
+    * Rigge, M., C. Homer, L. Cleeves, D. K. Meyer, B. Bunde, H. Shi, G. Xian,
+      S. Schell, and M. Bobo. 2020. Quantifying western U.S. rangelands as
+      fractional components with multi-resolution remote sensing and in situ
+      data. Remote Sensing 12.
+      [doi:10.3390/rs12030412](https://doi.org/10.3390/rs12030412)
+
+    * Rigge, M., C. Homer, H. Shi, D. Meyer, B.
+      Bunde, B. Granneman, K. Postma, P. Danielson, A. Case, and G. Xian. 2021.
+      Rangeland Fractional Components Across the Western United States
+      from 1985 to 2018. Remote Sensing 13:813.
+      [doi:10.3390/rs13040813](https://doi.org/10.3390/rs13040813)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'climate_change',
+    'disturbance',
+    'landsat_derived',
+    'nlcd',
+    'rangeland',
+    'trends',
+  ],
+  providers: [
+    ee.producer_provider(
+      'United States Geological Survey and Bureau of Land Management',
+      'https://www.mrlc.gov/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(
+      -125.0683594, 28.45903302, -101.0742188, 49.32512199,
+      '1985-01-01T00:00:00Z',
+      '2021-12-31T00:00:00Z'),
+  'gee:interval': {
+    type: 'cadence',
+    unit: 'year',
+    interval: 1,
+  },
+  summaries: {
+    gsd: [
+      30,
+    ],
+    'eo:bands': [
+      {
+        name: 'Break Point Count',
```

Comment:
These band names need to match what is in the data.

```
var dataset = ee.Image('USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/TRENDS');
print(dataset.bandNames());
```

Gives:

```
0: annual_herbaceous_break_point
1: bare_ground_break_point
2: herbaceous_break_point
3: litter_break_point
4: sagebrush_break_point
5: shrub_break_point
6: non_sagebrush_shrub_break_point
7: perennial_herbaceous_break_point
8: tree_break_point
9: annual_herbaceous_linear_model_slope
10: bare_ground_linear_model_slope
11: herbaceous_linear_model_slope
12: litter_linear_model_slope
13: sagebrush_linear_model_slope
14: shrub_linear_model_slope
15: non_sagebrush_shrub_linear_model_slope
16: perennial_herbaceous_linear_model_slope
17: tree_linear_model_slope
18: annual_herbaceous_linear_model_pvalue
19: bare_ground_linear_model_pvalue
20: herbaceous_linear_model_pvalue
21: litter_linear_model_pvalue
22: sagebrush_linear_model_pvalue
23: shrub_linear_model_pvalue
24: non_sagebrush_shrub_linear_model_pvalue
25: perennial_herbaceous_linear_model_pvalue
26: tree_linear_model_pvalue
27: annual_herbaceous_most_recent_break_point
28: bare_ground_most_recent_break_point
29: herbaceous_most_recent_break_point
30: litter_most_recent_break_point
31: sagebrush_most_recent_break_point
32: shrub_most_recent_break_point
33: non_sagebrush_shrub_most_recent_break_point
34: perennial_herbaceous_most_recent_break_point
35: tree_most_recent_break_point
36: total_change_intensity_index
```

Reply:
Yeah, those 37 are the correct ones. Do you have a script to update those so I don't have to replace them all? I know we updated the "year" band names, but we must not have yet for this one. Thanks

Reply:
https://gist.github.com/schwehr/3ef1c32fbdb13b51137acba79517d65e might have it, but make sure it's correct. You will likely need to modify the script.

Reply:
Should be fixed now. 37 areas updated.